services/slam/server_test.go

```diff
@@ -159,6 +159,25 @@ func TestWorkingServer(t *testing.T) {
 		test.That(t, respInfo.LastMapUpdate.AsTime(), test.ShouldResemble, timestamp)
 	})
 
+	t.Run("working GetProperties", func(t *testing.T) {
+		prop := slam.Properties{
+			CloudSlam:   false,
+			MappingMode: slam.MappingModeNewMap,
+		}
+		injectSvc.PropertiesFunc = func(ctx context.Context) (slam.Properties, error) {
+			return prop, nil
+		}
+
+		reqInfo := &pb.GetPropertiesRequest{
+			Name: testSlamServiceName,
+		}
+
+		respInfo, err := slamServer.GetProperties(context.Background(), reqInfo)
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, respInfo.CloudSlam, test.ShouldResemble, prop.CloudSlam)
+		test.That(t, respInfo.MappingMode, test.ShouldResemble, pb.MappingMode_MAPPING_MODE_CREATE_NEW_MAP)
```

Comment:
Why does this need to be should resemble?

Reply:
Resolved